spec/features/token_authentication_spec.rb

```diff
@@ -145,7 +145,7 @@
 
   scenario 'requesting token a second time after 3 hours sends different token url in email' do
     first_token_url = nil
-    Timecop.freeze(Time.now - 3.hours) do
+    Timecop.freeze(Time.now - 24.hours) do
```

Comment:
No need to change this now, but for future reference, ActiveSupport now has built-in helpers to do this.  You no longer need `Timecop`.

http://api.rubyonrails.org/v4.2.8/classes/ActiveSupport/Testing/TimeHelpers.html